gobblin-runtime/src/main/java/org/apache/gobblin/runtime/spec_catalog/FlowCatalog.java

```diff
@@ -457,4 +458,15 @@ public void remove(URI uri, Properties headers, boolean triggerListener) {
   public Object getSyncObject(String specUri) {
     return this.specSyncObjects.getOrDefault(specUri, null);
   }
+
+  public Spec getSpecFromStore(String specUri) {
```

Comment:
We should have the same name of the method, i.e. getSpecs(String uri) throws SpecNotFoundException